lib/chai/core/assertions.js

```diff
@@ -1765,6 +1765,12 @@ module.exports = function (chai, _) {
       , obj = flag(this, 'object')
       , ssfi = flag(this, 'ssfi');
 
+    if (typeof name !== 'string' && isNested) {
+      var msgPrefix = flag(this, 'message')
```

Comment:
We don't need to define `msgPrefix` here; the value of `flag(this, 'message')` is already stored in the `flagMsg` variable from line 1764.